golang/trace/zipin_tracereporter_test.go

```diff
@@ -60,11 +61,12 @@ func TestBuildZipkinSpan(t *testing.T) {
 			Port:        8888,
 			ServiceName: "testServer",
 		},
-		Name:        "test",
-		Id:          uint64ToBytes(span.SpanID()),
-		ParentId:    uint64ToBytes(span.ParentID()),
-		Annotations: buildZipkinAnnotations(annotations),
-		Debug:       false,
+		Name:              "test",
+		Id:                uint64ToBytes(span.SpanID()),
+		ParentId:          uint64ToBytes(span.ParentID()),
+		Annotations:       buildZipkinAnnotations(annotations),
+		BinaryAnnotations: buildBinaryAnnotations(binaryAnnotations),
+		Debug:             false,
```

Comment:
can you delete all the Debug lines - they will default to False anyway, and I don't think they're relevant